### PR TITLE
feat(md): allowed times constraints - add times group and fixed timezone

### PR DESCRIPTION
**Pull request by ranihorev**
image

### CODE REVIEW

File: app/scripts/modules/core/src/managed/constraints/AllowedTimes.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.DeploymentWindow {
+  margin-top: 4px;
+  padding-left: 16px;
+}
```
Comment (caseyhebebrand) [inline app/scripts/modules/core/src/managed/constraints/AllowedTimes.less:4]: You can remove this file and use the classnames `sp-margin-xs-top sp-padding-l-left` instead.

See the [styleguide reference](https://github.com/spinnaker/styleguide/blob/master/src/styles/spacing.css) for more info.

Comment (ranihorev) [reply app/scripts/modules/core/src/managed/constraints/AllowedTimes.less:4]: I was wondering about the source of these classes! thanks
btw, feels like we could use [tailwindcss](https://tailwindcss.com/). It's really awesome

Comment (ranihorev) [reply app/scripts/modules/core/src/managed/constraints/AllowedTimes.less:4]: Fixed, thanks!

**Review by caseyhebebrand — APPROVED** [review]
LGTM, thanks for writing tests!

**Pull request merged**